manifest.yaml

```diff
@@ -72,16 +72,12 @@ rules:
   - ""
   resources:
   - configmaps
```

Comment:
I'm sorry but I don't think that you understand that this is a generated file.

The role comes from the leader_election_role.yaml. It's a standard, generated file by Kubebuilder.

https://github.com/kubernetes-sigs/controller-tools/issues/401

Perhaps kubebuilder has a bug?